components/screens/ShoppingCart.js

```diff
@@ -16,15 +66,42 @@ function ShoppingCart({navigation}) {
 }
 
 const styles = StyleSheet.create({
-    container: {
+    container: {        
         flex: 1,
+        paddingTop: 40,
     },
     cart_container: {
-      flex: 1,  
-      backgroundColor: '#fff',
-      alignItems: 'center',
-      justifyContent: 'center',
+        alignItems: 'center',
+    },
+    product_container: {
+        flex: 1,
+        flexDirection: 'row',
+        justifyContent:'space-between',
+    },
+    product_image: {
+        height: 70,
+        width: 70,
+    },
+    product_name: {
+        fontSize: 20,
+    },
+    product_price: {
+        fontWeight: 'bold',
+        fontSize: 15,
```

Comment:
somewhere you have `fontSize` defined as `15` and somewhere you have defined as `20` 
can I know the reason for it?